src/default/DefaultWrap.js

```diff
@@ -27,12 +27,14 @@ class DefaultWrap extends SharedWrapMixin {
     this.fetchActiveResources();
   }
 
-  render() {
+  render(props) {
     const { config, matchStats, setStats, theme } = this.state;
 
     return (
       <div className="App" style={{ width: '1920px', height: '1080px', overflow: 'hidden' }}>
-        { matchStats && setStats && config && theme &&
+        { !(this.props.shouldFetchMatch && !matchStats)
```

Comment:
Oh-- is this the logic you were working out on the whiteboard? I see why this was tricky, ha.